keyboards/linworks/favepada/info.json

```
@@ -0,0 +1,105 @@
+{
+    "manufacturer": "Lx3",
+    "keyboard_name": "favepada",
+    "maintainer": "keydler",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "cols": ["A14", "A15", "B3", "B4"],
+        "rows": ["A9", "A8", "B15", "B14", "B13", "B12"]
+    },
+    "processor": "STM32F072",
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "layout": [
+            { "flags": 1, "matrix": [5, 2], "x": 149, "y": 59 },
+            { "flags": 1, "matrix": [5, 0], "x": 0, "y": 59 },
+            { "flags": 1, "matrix": [4, 0], "x": 0, "y": 51 },
+            { "flags": 1, "matrix": [4, 1], "x": 74, "y": 51 },
+            { "flags": 1, "matrix": [4, 2], "x": 149, "y": 51 },
+            { "flags": 1, "matrix": [4, 3], "x": 224, "y": 51 },
+            { "flags": 1, "matrix": [3, 2], "x": 149, "y": 38 },
+            { "flags": 1, "matrix": [3, 1], "x": 74, "y": 38 },
+            { "flags": 1, "matrix": [3, 0], "x": 0, "y": 38 },
+            { "flags": 1, "matrix": [2, 0], "x": 0, "y": 26 },
+            { "flags": 1, "matrix": [2, 1], "x": 74, "y": 26 },
+            { "flags": 1, "matrix": [2, 2], "x": 149, "y": 26 },
+            { "flags": 1, "matrix": [2, 3], "x": 224, "y": 26 },
+            { "flags": 1, "matrix": [1, 3], "x": 224, "y": 13 },
+            { "flags": 1, "matrix": [1, 2], "x": 149, "y": 13 },
+            { "flags": 1, "matrix": [1, 1], "x": 74, "y": 13 },
+            { "flags": 9, "matrix": [1, 0], "x": 0, "y": 13 },
+            { "flags": 2, "matrix": [0, 0], "x": 0, "y": 0 },
+            { "flags": 2, "matrix": [0, 1], "x": 74, "y": 0 },
+            { "flags": 2, "matrix": [0, 2], "x": 149, "y": 0 },
+            { "flags": 2, "matrix": [0, 3], "x": 224, "y": 0 },
+            { "flags": 4, "x": 224, "y": 5 },
+            { "flags": 4, "x": 149, "y": 5 },
+            { "flags": 4, "x": 74, "y": 5 },
+            { "flags": 4, "x": 0, "y": 5 },
+            { "flags": 4, "x": 0, "y": 18 },
+            { "flags": 4, "x": 0, "y": 31 },
+            { "flags": 4, "x": 0, "y": 43 },
+            { "flags": 4, "x": 0, "y": 56 },
+            { "flags": 4, "x": 0, "y": 64 },
+            { "flags": 4, "x": 74, "y": 64 },
+            { "flags": 4, "x": 149, "y": 64 },
+            { "flags": 4, "x": 224, "y": 64 },
+            { "flags": 4, "x": 224, "y": 56 },
+            { "flags": 4, "x": 224, "y": 43 },
+            { "flags": 4, "x": 224, "y": 31 },
+            { "flags": 4, "x": 224, "y": 18 }
+        ],
+        "max_brightness": 120
+    },
+    "rgblight": {
+        "led_count": 37
+    },
```

1. RGB_MATRIX led count is in `config.h`
2. rgblight is disabled
3. WS2812 pin migrated from `config.h`
```suggestion
    "ws2812": {
        "pin": "B1"
    },
```